catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,82 +12,80 @@
  *
  **/
 -->
-
+<!--
+=============================================================
+Blueprint.xml
+Implementation details
+=============================================================
+-->
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
-           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
-           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"
-           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-
-    <bean id="queryTemplateMetacardType"
-          class="org.codice.ddf.catalog.ui.forms.data.QueryTemplateType"/>
-
-    <bean id="attributeGroupMetacardType"
-          class="org.codice.ddf.catalog.ui.forms.data.AttributeGroupType"/>
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.0.0">
```

Comment:
what's the motivation of going to 1.0.0 vs 1.2.0 ?

Reply:
Nice catch. Version changes were not intentional. While we're still on core blueprint `1.0.0` I can verify and fix the `ext` version and `cm` version to what they were before. 
https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L16-L17